resources/vault-client_test.go

```diff
@@ -1,11 +1,17 @@
 package resources
```

Comment:
Can't find a more official file naming convention yet, but [this one](https://www.quora.com/What-is-the-best-naming-convention-of-keeping-source-files-in-a-programming-language) suggests just using underscore, which I think would look nicer also lol.